src/runner/bootstrapper.js

```diff
@@ -31,15 +46,35 @@ export default class Bootstrapper {
         return flatten(browserInfo);
     }
 
-    _createConnectionFromBrowserInfo (browserInfo) {
-        if (browserInfo instanceof BrowserConnection)
-            return browserInfo;
+    _createAutomatedConnections (browserInfo) {
+        if (!browserInfo)
+            return [];
+
+        var browserConnections = [];
+
+        for (var i = 0; i < this.concurrency; i++) {
+            var nextConnections = browserInfo.map(browser => new BrowserConnection(this.browserConnectionGateway, browser));
 
-        return new BrowserConnection(this.browserConnectionGateway, browserInfo);
+            browserConnections = zipWith(browserConnections, nextConnections, (prev, next) => {
+                if (!Array.isArray(prev))
+                    return [next];
+
+                return prev.concat(next);
+            });
+        }
+
+        return browserConnections;
```

Comment:
There is nice `times` function:
```js
return browserInfo
    .map(browser => times(concurrency, () => new BrowserConnection(this.browserConnectionGateway, browser)));
```
What do you think about it? \cc @AlexanderMoskovkin